src/lib.rs

```diff
@@ -188,6 +188,8 @@ pub struct InitOptions {
 pub fn init(options: &InitOptions) -> Result<ZWaveManager,()> {
     let mut ozw_options = try!(options::Options::create("./config/", "", "--SaveConfiguration true --DumpTriggerLevel 0 --ConsoleOutput false"));
 
+    try!(options::Options::add_option_int(&mut options, "PollInterval", 60000)); //enable polling each 60 seconds
+    try!(options::Options::add_option_bool(&mut options, "IntervalBetweenPolls", true));
```

Comment:
if we add these options it will make all updates very slow, as soon as we have more than 1 device. This can be good for production, but not while we're developing now.

Ultimately we'll want this to be configurable somehow. I don't know yet how we can do this -- maybe exposing some values of our own ?